hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -40,32 +140,48 @@ def perform_enque_for_review(match_message: MatchMessage):
     logger.debug("enqued for review")
 
 
-possible_actions = {
-    "WRITEBACK_IN_REVIEW": perform_writeback_in_review,
-    "ENQUE_FOR_REVIEW": perform_enque_for_review,
-}
+class LabelWithConstraints(Label):
+    _KEY_CONSTRAINT = "KeyConstraint"
+
+    def __init__(self, value: str):
+        super(LabelWithConstraints, self).__init__(self._KEY_CONSTRAINT, value)
 
 
-class ActionLabel(Label):
-    def __init__(self, key: str, value: str):
-        if key != "Action":
-            raise Exception("ActionLabels must have a key Action")
+class ActionLabel(LabelWithConstraints):
+    _KEY_CONSTRAINT = "Action"
 
-        if value not in possible_actions.keys():
-            raise Exception(f"'%s' is not a valid Action" % value)
 
-        super(self.__class__, self).__init__(key, value)
+class ThreatExchangeReactionLabel(LabelWithConstraints):
+    _KEY_CONSTRAINT = "ThreatExchangeReaction"
+
+
+@dataclass
+class Action:
```

Comment:
Again, I think we might want to remove this if we're not doing the priority

Reply:
@jeberl I have a feeling that the concept of priority and superseded_by is probably v1 (so, "coming soon to a codebase near you"). What's our standard on leaving stubs / reference points in code? I'm OK with deleting, or maybe commenting things out? Let's talk in the AM.